packages/core-data/src/hooks/constants.ts

```diff
@@ -4,3 +4,13 @@ export const enum Status {
 	Error = 'ERROR',
 	Success = 'SUCCESS',
 }
+
+export const EMPTY_ARRAY = [];
```

Comment:
Fair point. Just thought it was a new convention we were working towards and I came across these, so thought it would be a nice quick refactor to make things more consistent. If more devs see it as a step in the wrong direction, we can close this one.

Reply:
`EMPTY_ARRAY` and `EMPTY_OBJECT` and the like are special-cases regardless, because those are incidental details covering over the fact that all literal objects in JS have unique references, so things that want shallow-compare have to use the same one in comparison. within a module these are essentially free, but once we start building that coupling between modules we start pulling in overhead and concerns that really don't belong there for such a tiny optimization.